MapleServer2/Types/Player.cs

```diff
@@ -398,15 +398,22 @@ public void UpdateSocials()
         }
     }
 
-    public void Warp(int mapId, CoordF? coord = null, CoordF? rotation = null, long instanceId = -1)
+    public void Warp(int mapId, CoordF? coord = null, CoordF? rotation = null, long instanceId = -1, bool setReturnData = true)
     {
-        if (MapMetadataStorage.GetMetadata(mapId)?.Property.IsTutorialMap ?? true)
+        bool? isTutorialMap = MapMetadataStorage.GetMetadata(mapId)?.Property.IsTutorialMap;
```

Comment:
Use the metadata as nullable here, then on line 410, use metadata.Property.IsTutorialMap

Reply:
aight!